src/ontology/mondo-edit.obo

```diff
@@ -556614,7 +556614,7 @@ is_a: MONDO:0020573 {source="OMIM:617921", source="https://orcid.org/0000-0001-5
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0060670 {source="MONDO:CLINGEN"}
 relationship: excluded_subClassOf MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 relationship: excluded_subClassOf MONDO:0005144 {source="OMIM:617921", source="https://orcid.org/0000-0001-5208-3432"} ! familial amyotrophic lateral sclerosis
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6323 {source="OMIM:617921"} ! KIF5A
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6323 ! KIF5A
```

Comment:
removed because omim tags this as a mutation that contributes to susceptibility to multifactorial disorders https://omim.org/entry/617921 and https://omim.org/help/faq#1_6

Reply:
This OMIM should have been kept. We have patterns using gene association for inherited susceptibilities. We should discuss/review what we do for susceptibilities.